Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -223,17 +225,27 @@ open class AVFoundationPlayback: Playback {
             }
         }
     }
-
+    
+    private var shouldLoop: Bool {
+        return options.bool(kLoop)
+    }
+    
+    private func newAVQueuePlayer(with item: AVPlayerItem) -> AVQueuePlayer {
+        let player = AVQueuePlayer(items: [item])
+        playerLooper = AVPlayerLooper(player: player, templateItem: item)
```

Comment:
I think this method's name leads to a misunderstanding of what it does. I mean, it not just instantiates a new `AVQueuePlayer` but it also sets the `playerLooper` variable.
In my opinion, the simplest solution would be to rename the method to something like `setupQueuePlayer` as it may include more steps than just creating a new instance of `AVQueuePlayer`. 

What do you guys think about that?